src/components/views/gallery/main-banner/main-banner.component.tsx

```diff
@@ -22,7 +22,7 @@ function MainBanner() {
         }}
       />
       <Container>
-        <h1 className={styles.title}>{t("mainBanner.title")}</h1>
+        <h1 className={styles.title}>{t("galleryBanner.title")}</h1>
```

Comment:
tłumaczenie weź z common i zmień z "galleryBanner.title" na "gallery" (tłumaczenie jest na masterze najnowszym)